src/youtube-player/youtube-player.ts

```diff
@@ -71,6 +74,29 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy {
 
   private _videoId = new EventEmitter<string | undefined>();
 
+  /** Height of video player */
+  get height(): number | undefined {
+    return this._height;
+  }
+
+  @Input() set height(height: number | undefined) {
```

Comment:
We prefer to put `@Input` on the getter. We also put `@Input` on its own line, but then also condense single-line getters like
```ts
@Input()
get thumbLabel(): boolean { return this._thumbLabel; }
```